index.html

```diff
@@ -0,0 +1,177 @@
+               neque quisquam fugit officiis voluptates magnam amet ea ex molestiae quibusdam omnis
+            </p>
+    
+            <h2 class="time black">2022.10.15(THU) ~ 16(FRI)</h2>
+            <h3 class="venue black">@ Lekki phase 2, Lagos Nigeria</h3>
+        </div>    
+        <section id="header-about-section"> 
+            <h2 class="red">BEATZ Station Music streaming</h2>
+            <div>
+                <p id="about-main-content" class="black">
+                    Lorem ipsum dolor sit, amet consectetur adipisicing elit. 
+                    Aliquid veritatis reprehenderit voluptatem cum libero omnis, quisquam quas suscipit
+                    ratione quibusdam incidunt non ducimus dicta minima consectetur vitae inventore facilis nulla?
+                </p>
+            </div>
+            <div class="visit-link">
+                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit.</p>
+                <p><strong>ccsummit2022@ccworldwide.org</strong></p>
+            </div>
+        </section>
+    </header>
+    <div id="bodi-homepage-section">
+        <div class="latest-songs">
+            <h2 class="gray">New Releases</h2>
+            <hr class="line-border">
+        </div>
+        <div class="songs-list ">
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <button id="selectsong" type="button">Jump on the Beat</button>
+        </div>
```

Comment:
- [ ] Please center the button :
<img width="456" alt="Screenshot 2022-03-11 at 18 35 59" src="https://user-images.githubusercontent.com/77038610/157887952-6cf4bc1a-1ead-4877-b8b9-15916ffea747.png">
You can simply do that by using flexbox `align-items` property:

```
#selectsong {
    margin: 40px 0 40px 0;
}
.songs-list {
   align-items: center;
}
```